Dockerfile.local

```diff
@@ -0,0 +1,9 @@
+FROM gcr.io/distroless/static:debug
+
+WORKDIR /
+COPY bin/k8s .
```

Comment:
one additional thought I've had around this -- do we want to ensure somehow that the binaries being copied into the image are not stale? maybe `build-container` should run `make build` first?

Reply:
Yes, `build-container` (I changed to `build-local-container`) target is depend on `build` target